oneplus/com_views.py

```diff
@@ -273,6 +273,7 @@ def post():
 
                 _message.save()
             request.session["state"]["chat_page_max"] += 1
+            return redirect("com.chat", chatid=chatid)
```

Comment:
Just checking if this was supposed to be addd. The commit message doesn't match the changes in this commit